cleanlab/classification.py

```diff
@@ -404,105 +556,21 @@ class label.
-                )
-
-            self.sample_weight = np.ones(np.shape(labels_cleaned))
-            for k in range(self.K):
-                sample_weight_k = 1.0 / max(self.noise_matrix[k][k], 1e-3)  # clip sample weights
-                self.sample_weight[labels_cleaned == k] = sample_weight_k
-
-            if self.verbose:
-                print("Fitting final model on the clean data ...")
-            self.clf.fit(
-                x_cleaned, labels_cleaned, sample_weight=self.sample_weight, **self.clf_final_kwargs
-            )
-        else:
-            if self.verbose:
-                print("Fitting final model on the clean data ...")
-            # This is less accurate, but best we can do if no sample_weight.
-            self.clf.fit(x_cleaned, labels_cleaned, **self.clf_final_kwargs)
-
-        return self.clf
-
-    def predict(self, *args, **kwargs):
-        """Returns a vector of predictions.
-
-        Parameters
-        ----------
-        X : :obj:`np.array` of shape (n, m)
-          The test data as a feature matrix."""
-
-        return self.clf.predict(*args, **kwargs)
-
-    def predict_proba(self, *args, **kwargs):
-        """Returns a vector of probabilities P(true_label=k)
-        for each example in X.
+            print(f"Identified {np.sum(self.label_issues_mask)} datapoints with label issues.")
 
-        Parameters
-        ----------
-        X : :obj:`np.array` of shape (n, m)
-          The test data as a feature matrix."""
-
-        return self.clf.predict_proba(*args, **kwargs)
-
-    def score(self, X, y, sample_weight=None):
-        """Returns the clf's score on a test set X with labels y.
-        Uses the model/clf's default scoring function.
-
-        Parameters
-        ----------
-        X : :obj:`np.array` of shape (n, m)
-          The test data as a feature matrix.
-
-        y : :obj:`np.array<int>` of shape (n,) or (n, 1)
-          The test classification labels as an array.
-
-        sample_weight : :obj:`np.array<float>` of shape (n,) or (n, 1)
-          Weights each example when computing the score / accuracy."""
-
-        if hasattr(self.clf, "score"):
-
-            # Check if sample_weight in clf.score(). Not compatible with Python 2.
-            if "sample_weight" in inspect.getfullargspec(self.clf.score).args:
-                return self.clf.score(X, y, sample_weight=sample_weight)
-            else:
-                return self.clf.score(X, y)
-        else:
-            return accuracy_score(
-                y,
-                self.clf.predict(X),
-                sample_weight=sample_weight,
-            )
+        return self.label_issues_mask
 
     def get_label_issues(self):
-        """Accessor. Returns `self.label_issues_mask` computed via `filter.find_label_issues()`"""
+        """Accessor. Returns `self.label_issues_mask` if previously already computed."""
 
         return self.label_issues_mask
```

Comment:
I was thinking it'd be nice to raise a warning and print a message if the user calls `get_label_issues()` when `self.label_issues_mask` is None. 

Something like: `"label_issues_mask has not been computed yet! To compute label_issues_mask, please run find_label_issues() or fit() first."`. This can be a separate PR though.

Reply:
+1 Good idea. I'd add this in this PR. Its just one if statement and a warnings import. Seems useful to educate users.

Reply:
implemented.